packages/experiments-realm/blog-app.gts

```diff
@@ -90,26 +92,31 @@ interface SidebarFilter {
   isCreateNewDisabled?: boolean;
   cardRef?: ResolvedCodeRef;
   query?: Query;
+  isSortDisabled?: boolean;
+  showAdminData?: boolean;
 }
 const FILTERS: SidebarFilter[] = [
   {
     displayName: 'Blog Posts',
     icon: BlogPostIcon,
     cardTypeName: 'Blog Post',
     createNewButtonText: 'Post',
+    showAdminData: true,
   },
   {
     displayName: 'Author Bios',
     icon: AuthorIcon,
     cardTypeName: 'Author Bio',
     createNewButtonText: 'Author',
+    isSortDisabled: true,
```

Comment:
Instead of `isSortDisabled`, maybe we should have a `sortOptions` prop here. If provided, sort is rendered with those options. If not, no sort UI is rendered.

Reply:
That's what I initially did, but that implies sortOptions might be different per filter and we have to reset or change selectedSort as the filter changes so as not to do a halfway-done implementation. I can change back to it though.

Reply:
Your call -- just seemed duplicative to me, but I understand your point.